lib/auth/auth.go

```diff
@@ -2312,9 +2311,13 @@ func (a *Server) isMFARequired(ctx context.Context, checker services.AccessCheck
 		// If at least one node requires MFA, we'll catch it.
 		for _, n := range matches {
 			err := checker.CheckAccessToServer(t.Node.Login, n, services.AccessMFAParams{AlwaysRequired: false, Verified: false})
-			if err != nil {
+			if err == nil {
+				continue
+			} else if errors.Is(err, services.ErrSessionMFARequired) {
 				noMFAAccessErr = err
 				break
+			} else {
+				log.Debugf("ignoring unrelated error while checking access to server: %v", err)
```

Comment:
Probably unnecessary, any error here should re-emerge during the real server access attempt.

Reply:
Fair enough, done